thirdparty/build-thirdparty.sh

```diff
@@ -787,7 +787,7 @@ build_bitshuffle() {
 build_croaringbitmap() {
     avx_flag=
     if [ ! -z "$USE_AVX2" -a "$USE_AVX2" -eq 0 ];then
-        echo "FORCE disable AVX for croaringbitmap"
+        echo "set USE_AVX2=$USE_AVX2 to FORCE disable AVX2 in croaringbitmap"
```

Comment:
This prompt seems confusing, is it `disable` or `enable`?

Reply:
Only after set USE_AVX2=0, it will reach here.  It means disable AVX2.
Otherwise we will open AVX2 and there is no message printed.